### PR TITLE
Add ocp-power-automation/ci-monitoring-automation project

### PR DESCRIPTION
**Pull request by yussufsh**

### CODE REVIEW

**Comment by yussufsh** [comment]
@Shilpa-Gokul

File: config/prow/plugins.yaml [hunk 1]
```diff
@@ -271,3 +271,8 @@ external_plugins:
     events:
       - issue_comment
       - pull_request
+  ocp-power-automation/ci-monitoring-automation:
+  - name: mergable
```
Comment (Shilpa-Gokul) [inline config/prow/plugins.yaml:275]: Is that mergeable?

Comment (Rajalakshmi-Girish) [reply config/prow/plugins.yaml:275]: Yes, it was a typo with the previous repo at L269 as well.
I think it has nothing to do with this plugin.
I shall get back.

**Comment by mkumatag** [comment]
@yussufsh please create PR from your fork

**Comment by mkumatag** [comment]
@yussufsh this plugin was written only to handle that +2, please avoid adding this config if you don't have +2 configured the repo

**Pull request closed without merging**